isso/migrate.py

```diff
@@ -225,9 +225,18 @@ def migrate(self):
         progress.finish("{0} threads, {1} comments".format(
             len(items) - skip, self.count))
 
+    def _process_comment_content(self, text):
+        # WordPress comment text renders a single newline between two blocks of
+        # text as a <br> tag, so convert to double newlines when importing
+        # (Otherwise multiple blocks of text separated by single newlines are
+        # all shown as one long line.)
+        text = re.sub(r'(?!^\n)\n(?!^\n)', '\n\n', text, 0)
```

Comment:
```suggestion
        text = re.sub(r'(?!^\n)\n(?!^\n)', '  \n', text, 0)
```

Reply:
I meant literally that you should insert two spaces at the end of the line. E.g. this:
```markdown
This is a text with 2 trailing spaces  
Next line

Two newlines should not be affected.
```
will render as
```html
<p>This is a text with 2 trailing spaces<br>
Next Line</p>
<p>Two newlines should not be affected</p>
```

Reply:
@ix5 oh snap, sorry I totally misread your first suggestion. And I clearly need to read the Markdown spec more often!

Updated, thanks for re-explaining.